Formula/bnd.rb

```diff
@@ -0,0 +1,50 @@
+class Bnd < Formula
+  desc "the Swiss Army Knife for OSGi bundles"
+  homepage "http://bnd.bndtools.org/"
+  url "https://search.maven.org/remotecontent?filepath=biz/aQute/bnd/biz.aQute.bnd/3.3.0/biz.aQute.bnd-3.3.0.jar"
+  sha256 "b6b68dfcd0f5ba767a202bf35eb3eb964c63e679e8217dd514dac807e6cedee8"
+
+  bottle :unneeded
+
+  def install
+    prefix.install "biz.aQute.bnd-#{version}.jar"
+
+    bin.write_jar_script prefix/"biz.aQute.bnd-#{version}.jar", "bnd"
+  end
+
+  test do
+    # Simple version output check...
+    assert_match /^#{version}/, shell_output("#{bin}/bnd version")
+
+    # Test bnd by resolving a launch.bndrun file against a trivial index.
+    # First write the index...
+    (testpath/"index.xml").write <<-EOS.undent
+      <?xml version="1.0" encoding="utf-8"?>
+      <repository increment="0" name="Untitled" xmlns="http://www.osgi.org/xmlns/repository/v1.0.0">
+        <resource>
+          <capability namespace="osgi.identity">
+            <attribute name="osgi.identity" value="org.apache.felix.gogo.runtime"/>
+          </capability>
+          <capability namespace="osgi.content">
+            <attribute name="osgi.content" value="baad835c6fa65afc1695cc92a9e1afe2967e546cae94d59fa9e49b557052b2b1"/>
+            <attribute name="url" value="org.apache.felix.gogo.runtime-1.0.0.jar"/>
```

Comment:
May want to consider making some of these valuables variables rather than hard-coding in two places.